src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -2,17 +2,31 @@
 
 
 import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.IndexerVelcroDown;
+import org.usfirst.frc4904.robot.commands.IndexerVelcroHeld;
+import org.usfirst.frc4904.robot.commands.IndexerVelcroRelease;
+import org.usfirst.frc4904.robot.commands.IndexerVelcroUp;
 import org.usfirst.frc4904.standard.humaninput.Operator;
+import org.usfirst.frc4904.standard.custom.controllers.CustomJoystick;
+
 
 public class DefaultOperator extends Operator {
 	public DefaultOperator() {
 		super("DefaultOperator");
 	}
 
 	public DefaultOperator(String name) {
+	
 		super(name);
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button1.onlyWhileHeld(new IndexerVelcroHeld());
```

Comment:
These commands should not be run as onlyWhileHeld because we do not need to keep setting a piston over and over again if the button is held down. Use whenPressed instead